### ISSUE TITLE
Window theme is being generated but not updating

### ISSUE THREAD

**Issue by CrashyBang** [deviantfero/wpgtk#21]
I am having to manually update the window theme using __ObConf__ before it updates.

After watching the video I believe it is meant to be automatic?

Cheers.

**Comment by deviantfero** [comment]
hi there, can you run wpg from a console and paste the output here? also try manually running
`openbox --reconfigure` or
`openbox --restart` if reconfigure does not work, it is suposed to be automatic though

**Comment by CrashyBang** [comment]
Hey @deviantfero that was quick :).

Okay so it thinks I do not have an openbox install... but I am indeed using openbox:

```
CURRENT WALL: /home/otis/.wallpapers/wallpaper.png
:: not installed by AUR
:: Setting rope.jpg as background
:: Done
FG: ab791f
BG: 6f4f14
CHANGED::OPENBOX
NO OPENBOX INSTALL!
FAILED TO CHANGE::TINT2 - BASE FILE DOES NOT EXIST
CHANGED::GTK2
CHANGED::GTK3
CHANGED::GTK3.20
CHANGED::ICONS
CURRENT GLYPH: 3b394f
NO OPTIONAL FILES DETECTED::NOT CHANGED
SUCCESS
```

**Comment by deviantfero** [comment]
I know what the issue is, the way i check for an openbox install, is checking if `~/.config/openbox/menu.xml` exists, which is the default openbox file for configuring the menu that gets displayed with the right click.

so for now run:
```sh
mkdir -p ~/.config/openbox && touch ~/.config/menu.xml
```
or in simpler terms, create a menu.xml in that directory, even if it's empty! please let me know if you already have that file before doing this.

**Comment by CrashyBang** [comment]
Hey @deviantfero awesome that has fixed it.

May I suggest checking for an openbox install using the default location of `/etc/xdg/openbox/` here you can check for the `menu.xml` or any of the other files stored here by default?
